lib/luwra/usertypes.hpp

```diff
@@ -217,7 +218,7 @@ struct Value<U*> {
 	 * \returns Number of values that have been pushed
 	 */
 	static inline
-	size_t push(State* state, const T* ptr) {
+	size_t push(State* state, T* ptr) {
```

Comment:
Was part of trying to use arbitrary functions: now that I'm using the macro, I'll revert this and the other line.